js/modules/types/contact.js

```diff
@@ -39,12 +44,17 @@ exports.parseAndWriteAvatar = upgradeAttachment => async (
   return parsedContact;
 };
 
-function parseContact(contact) {
+function parseContact(contact, options = {}) {
+  const { regionCode } = options;
+
+  const boundParsePhone = phoneNumber =>
+    parsePhoneItem(phoneNumber, { regionCode });
```

Comment:
Could we shorten this to `parsePhone`? It’s using regular function composition so `bound*` prefix seems redundant.

Reply:
It's definitely still bound to a value; it doesn't have to be an explicit `bind()` call.